### PR TITLE
Update README.md to include @types reference

### PR DESCRIPTION
**Pull request by mcrawshaw**
Adding reference to non-obviously named @types package.

### CODE REVIEW

**Comment by smbkr** [comment]
Great idea, this is a pretty pragmatic solution to finding a balance between peoples desire for type support, and the maintainer’s hesitance to add official support

**Comment by esetnik** [comment]
@mcrawshaw `@types` should be installed as dev dependencies. So `npm install -D`

**Comment by mcrawshaw** [comment]
@esetnik sorry, fixed.

File: README.md [hunk 1]
```diff
@@ -34,6 +34,14 @@ Install using [npm](https://www.npmjs.org/):
 npm install @koa/router
 ```
 
+## Typescript
+
+Although we do not support directly, there is an associated @types package.
```
Comment (fedyk) [inline README.md]: Here we can add an URL to types

```suggestion
Although we do not support directly, there is an associated [@types](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/koa__router) package.
```

**Comment by JacobMGEvans** [comment]
Is there any plan on adding the documentation from the old repository into the README.md or another file or just continue to reference the old repository.

**Comment by niftylettuce** [comment]
Conflicts would need fixed.  But I do not support nor want to recommend users to use TypeScript.  It's very bad for the community.

**Comment by puzzledbytheweb** [comment]
> Conflicts would need fixed. But I do not support nor want to recommend users to use TypeScript. It's very bad for the community.

Don't you think that it's worse for the community not to have that option? If the idea is to have as many users as possible I'd say that it would be a good idea to make usage as easy as possible for everyone regardless of the choice of tooling they have. But that's just my opinion

**Comment by niftylettuce** [comment]
I'm closing this.  I will not support TypeScript and do not recommend people to use it.  Unnecessary convolution.

**Pull request closed without merging**